piponic.py

```diff
@@ -115,6 +118,7 @@ def update_sensor_data(self):
         try:
             self.pH = self.adc_sensors.read_pH()
             self.leak = self.adc_sensors.read_leak()
+            self.water_level = self.water_level_sensor.read()
         except:
             print('Error ADC or I2C Error')
```

Comment:
Would be good to have a more accurate error message, saying that there is an error reading one of the sensors.

Reply:
yeah good idea.